### ISSUE TITLE
Why does spinbutton support aria-activedescendant?

### ISSUE THREAD

**Issue by JAWS-test** [w3c/aria#1350]
I cannot imagine a meaningful implementation of a [spinbutton](https://rawgit.com/w3c/aria/master/#spinbutton) with aria-activedescendant. This would only work if the input field of the spinbutton refers to an external list of possible items. Is that what we want? If so, which role should the items have (none or option in listbox?)

spinbutton inherits aria-activedescendant from the superclass role composit. The easiest solution would be to remove composite as superclass role

**Comment by jnurthen** [comment]
fixing this seems a little like busy work. I'm not saying it is correct but I feel our time might be better spent elsewhere.
Closing this. If someone wants to make a PR it will be considered.
